packages/pytest-simcore/src/pytest_simcore/helpers/rawdata_fakers.py

```diff
@@ -33,7 +33,7 @@
 from simcore_postgres_database.models.users import users
 from simcore_postgres_database.webserver_models import GroupType, UserStatus
 
-STATES = [
+COMPUTATIONAL_SERVICE_STATES = [
```

Comment:
hmm. that is used in comp_tasks table. and in comp_tasks table all types of services are available.

Reply:
reverted